campaignion_action/campaignion_action.module

```diff
@@ -79,13 +79,20 @@ function campaignion_action_module_implements_alter(&$impl, $hook) {
     }
     $impl = $new_impl;
   }
-  if ($hook == 'webform_redirect_alter') {
+  $module = 'campaignion_action';
+  $move_to_front = function () use (&$impl, $module) {
```

Comment:
What’s `$module` for, it doesn’t seem to be used anywhere?

Reply:
I’ve removed it in a new commit.